spec/system/documents/edit_documents_spec.rb

```diff
@@ -52,6 +52,82 @@
       expect(page).to have_content("Public: Yes")
     end
 
+    it "allows for a document to be marked as not valid" do
+      click_link "Edit"
+
+      within("#validate-document") do
+        choose "No"
+      end
+
+      element = find(:css, "#document-invalidated-document-reason-field")
+      element.fill_in with: "BANANAS"
+
+      click_button("Save")
+
+      expect(page).to have_content("Invalid documents: 1")
+      expect(page).to have_content("BANANAS")
+    end
+
+    it "requires a comment if the document is marked as not valid" do
+      click_link "Edit"
+
+      within("#validate-document") do
+        choose "No"
+      end
+
+      element = find(:css, "#document-invalidated-document-reason-field")
```

Comment:
I left a comment above on the form - attaching label text, even if the css hides it, is better for accessibility and will also help Capybara find the input box by text rather than css